pkg/server/sessions/manager.go

```diff
@@ -42,7 +40,7 @@ type manager struct {
 	ticker     *time.Ticker
 	done       chan bool
```

Comment:
running, ticker and done may be prefixed with sessionGuard e.g. sessionGuardRunning, sessionGuardTicker sessionGuardDone. Or even better extract into it's own struct...

Reply:
or actually, it's the session manager itself and the concept of session guard is not needed... ?

Reply:
anyways, this are non-blocking changes